shell/platform/linux/fl_view.cc

```diff
@@ -491,6 +499,8 @@ static gboolean event_box_motion_notify_event(GtkWidget* widget,
     return FALSE;
   }
 
+  view->pointer_inside = TRUE;
```

Comment:
You can end up inside a window without a motion event (e.g. by switching windows with alt-tab without moving the mouse). This flag should be set by listening to the enter-notify-event signal.

Reply:
In my experience the enter-notify is something missing, i.e. when showing menus. I think most robust solution would be to use both motion-notify and enter-notify.